tests/steps/application_v2_steps.py

```diff
@@ -24,6 +24,8 @@ def operation_string_to_enum(operation):
         return transaction.OnComplete.NoOpOC
     elif operation == "create":
         return transaction.OnComplete.NoOpOC
+    elif operation == "create-and-optin":
```

Comment:
How did this come to be called "create-and-optin"?

Reply:
I added it when adding this cucumber test: https://github.com/algorand/algorand-sdk-testing/blob/7f06a6bd5fa352f87d62e108e2dc35bf05021343/features/integration/simulate.feature#L212

For legacy reasons, this step has "operations" which are the standard on completion options, and a "create" operation, which means noop but also don't include any app ID. I needed an app create that also opted in, so I chose to define another "operation" type that did that. Not the best solution but it was the easiest